internal/controllers/machinehealthcheck/machinehealthcheck_targets.go

```diff
@@ -34,6 +34,7 @@ import (
 	"sigs.k8s.io/cluster-api/util"
```

Comment:
General question. This PR will now set the conditions on the Machine, but MS/KCP don't clean them up. I assume we'll take care that this is implemented as well ahead of the release? Otherwise we can easily confuse folks

Reply:
Yes, I'm working on KCP cleanup part in parallel, and MS will be addressed too